spring-data-neo4j-rx/src/main/java/org/neo4j/springframework/data/core/schema/CypherGenerator.java

```diff
@@ -329,5 +348,16 @@ private MapProjection projectPropertiesAndRelationships(NodeDescription<?> nodeD
 	private static Condition conditionOrNoCondition(@Nullable Condition condition) {
 		return condition == null ? Conditions.noCondition() : condition;
 	}
+
+	private boolean hasInverse(RelationshipDescription relationshipDescription,
+		Collection<RelationshipDescription> relationships) {
```

Comment:
Please rename `relationships` to `processedRelationships`.

I wonder if we could avoid the loop somehow with a `contains`.